packages/react/src/generators/component-story/files/__componentFileName__.stories.__fileExt__

```diff
@@ -1,19 +1,14 @@
-<% if(usedKnobs && usedKnobs.length > 0) { %>import { <%= usedKnobs %> } from '@storybook/addon-knobs';<% } %>
-import React from 'react';
+import { Story, Meta } from '@storybook/react';
 import { <%= componentName %><% if ( propsTypeName && !isPlainJs ) { %>, <%= propsTypeName %> <% } %> } from './<%= componentFileName %>';
```

Comment:
I don't know if it makes sense in the context of NX, but starting from Storybook 6.2 there are these two new types `ComponentStory` and `ComponentMeta`, and they infer the props from a component when you write it like this:

```js
export default {
  title: 'MyComponent'
} as ComponentMeta<typeof MyComponent>;

const Template: ComponentStory<typeof MyComponent> = () => <.../>;
```

Meaning that there's no need to export props anymore just to satisfy Storybook.

However, I don't know how this template is used ,so if this is supposed to work with projects using older versions of Storybook, then ignore this suggestion!

```suggestion
import { ComponentStory, ComponentMeta } from '@storybook/react';
import { <%= componentName %> } from './<%= componentFileName %>';
```

Reply:
`ComponentStory` and `ComponentMeta` are introduced in [v6.3.0-alpha.22](https://github.com/storybookjs/storybook/releases/tag/v6.3.0-alpha.22) and they are not ported to v6.2.

Reply:
Good catch, my mistake!